### ISSUE TITLE
Can't install 0.14: directory of vendored discount seems to be corrupted in release in CPAN

### ISSUE THREAD

**Issue by polamjag** [sekimura/text-markdown-discount#29]
Thank you for maintaining this library <3

## What's happening

- Can't install 0.14.

### Some investigations

- I found that directory for vendored discount library seems to be corrupted.
  - Tree in GitHub contains bunch of source codes of discount: https://github.com/sekimura/text-markdown-discount/tree/73134d6245919e5b74532742e6ebc8bb98e019a2/discount-2.2.7
  - Tree in metacpan does not contain any source codes: https://metacpan.org/release/SEKIMURA/Text-Markdown-Discount-0.14/source/discount
- By the way, directory name for discount library is inconsistent between CPAN and Makefile.PL: `discount` in CPAN but `discount-2.2.7` in Makefile.PL
  - https://github.com/sekimura/text-markdown-discount/blob/73134d6245919e5b74532742e6ebc8bb98e019a2/Makefile.PL#L7
  - https://metacpan.org/release/SEKIMURA/Text-Markdown-Discount-0.14/source/Makefile.PL#L7

## Steps to reproduce

<details>
<summary>run `Text::Markdown::Discount` (click to expand logs)</summary>

```
$ cpan install Text::Markdown::Discount
<snip>
Configuring S/SE/SEKIMURA/Text-Markdown-Discount-0.14.tar.gz with Makefile.PL
Checking if your kit is complete...
Warning: the following files are missing in your kit:
        discount/amalloc.c
        discount/amalloc.h
        discount/basename.c
        discount/configure.inc
        discount/configure.sh
        discount/COPYRIGHT
        discount/CREDITS
        discount/Csio.c
        discount/css.c
        discount/cstring.h
        discount/docheader.c
        discount/dumptree.c
        discount/emmatch.c
        discount/flags.c
        discount/generate.c
        discount/github_flavoured.c
        discount/html5.c
        discount/INSTALL
        discount/main.c
        discount/Makefile.in
        discount/makepage.1
        discount/makepage.c
        discount/markdown.1
        discount/markdown.3
        discount/markdown.7
        discount/markdown.c
        discount/markdown.h
        discount/mkd-callbacks.3
        discount/mkd-extensions.7
        discount/mkd-functions.3
        discount/mkd-line.3
        discount/mkd2html.1
        discount/mkd2html.c
        discount/mkdio.c
        discount/mkdio.h.in
        discount/mktags.c
        discount/pgm_options.c
        discount/pgm_options.h
        discount/Plan9/markdown.1
        discount/Plan9/markdown.2
        discount/Plan9/markdown.6
        discount/Plan9/mkfile
        discount/Plan9/README
        discount/README
        discount/resource.c
        discount/setup.c
        discount/tags.c
        discount/tags.h
        discount/tests/autolink.t
        discount/tests/automatic.t
        discount/tests/backslash.t
        discount/tests/callbacks.t
        discount/tests/chrome.text
        discount/tests/code.t
        discount/tests/compat.t
        discount/tests/crash.t
        discount/tests/defects.t
        discount/tests/div.t
        discount/tests/dl.t
        discount/tests/embedlinks.text
        discount/tests/emphasis.t
        discount/tests/extrafootnotes.t
        discount/tests/flow.t
        discount/tests/footnotes.t
        discount/tests/functions.sh
        discount/tests/header.t
        discount/tests/html.t
        discount/tests/html5.t
        discount/tests/links.text
        discount/tests/linkylinky.t
        discount/tests/linkypix.t
        discount/tests/list.t
        discount/tests/list3deep.t
        discount/tests/misc.t
        discount/tests/pandoc.t
        discount/tests/para.t
        discount/tests/paranoia.t
        discount/tests/peculiarities.t
        discount/tests/pseudo.t
        discount/tests/reddit.t
        discount/tests/reparse.t
        discount/tests/schiraldi.t
        discount/tests/smarty.t
        discount/tests/snakepit.t
        discount/tests/strikethrough.t
        discount/tests/style.t
        discount/tests/superscript.t
        discount/tests/syntax.text
        discount/tests/tables.t
        discount/tests/tabstop.t
        discount/tests/toc.t
        discount/tests/xml.t
        discount/theme.1
        discount/theme.c
        discount/toc.c
        discount/tools/checkbits.sh
        discount/tools/cols.c
        discount/tools/echo.c
        discount/VERSION
        discount/version.c.in
        discount/xml.c
        discount/xmlpage.c
Please inform the author.
Generating a Unix-style Makefile
Writing Makefile for Text::Markdown::Discount
Writing MYMETA.yml and MYMETA.json
  SEKIMURA/Text-Markdown-Discount-0.14.tar.gz
  /usr/local/Cellar/perl/5.36.0/bin/perl Makefile.PL -- OK
Running make for S/SE/SEKIMURA/Text-Markdown-Discount-0.14.tar.gz
cp lib/Text/Markdown/Discount.pm blib/lib/Text/Markdown/Discount.pm
( cd discount-2.2.7; CC='cc -fPIC' sh configure.sh --with-fenced-code --with-dl=both; make )
/bin/sh: line 0: cd: discount-2.2.7: No such file or directory
sh: configure.sh: No such file or directory
( cd discount-2.2.7; CC='cc -fPIC' sh configure.sh --with-fenced-code --with-dl=both; make )
/bin/sh: line 0: cd: discount-2.2.7: No such file or directory
sh: configure.sh: No such file or directory
( cd discount-2.2.7; CC='cc -fPIC' sh configure.sh --with-fenced-code --with-dl=both; make )
/bin/sh: line 0: cd: discount-2.2.7: No such file or directory
sh: configure.sh: No such file or directory
( cd discount-2.2.7; CC='cc -fPIC' sh configure.sh --with-fenced-code --with-dl=both; make )
/bin/sh: line 0: cd: discount-2.2.7: No such file or directory
sh: configure.sh: No such file or directory
( cd discount-2.2.7; CC='cc -fPIC' sh configure.sh --with-fenced-code --with-dl=both; make )
<this error continues infinitely>
```

</details>

**Comment by sekimura** [comment]
Thanks for reporting the issue. I've updated the ext dirname to `discount-2.2.7` but forgot to update the manifest files. https://github.com/sekimura/text-markdown-discount/commit/eac211d80dd9702bc54c38c2bc8825ed1cbcc562 address the issue. The new version was uploaded and it should be indexed soon.

**Comment by polamjag** [comment]
Thank you for fixing! I've confirmed that problem is resolved in 0.16 (macOS and Linux).

All the problems I am aware of have been resolved, so I am closing this issue. I would appreciate it if you could reopen it if you have any problems.
